r/R/arrow-package.R

```diff
@@ -89,7 +89,8 @@ arrow_info <- function() {
   out <- list(
     version = packageVersion("arrow"),
     libarrow = arrow_available(),
-    options = opts[grep("^arrow\\.", names(opts))]
+    options = opts[grep("^arrow\\.", names(opts))],
+    runtime = runtime_info()
```

Comment:
Don't put this in the `out` object because you're going to reformat it and add it in below.